src/components/RadioGroup/examples/default.jsx

```diff
@@ -8,7 +8,7 @@ const style = {
 export default React.createClass({
 	render() {
 		return (
-			<RadioGroup>
+			<RadioGroup name='name'>
```

Comment:
That's a good point -- what do you think about using [_.uniqueId](https://lodash.com/docs/4.16.6#uniqueId) rather than our `random()` to generate these IDs? That would let me keep the example the same but also give consistent unique identifiers.